### ISSUE TITLE
Sony Xperia S (nozomi)

### ISSUE THREAD

**Issue by MichaelSp** [DsLNeXuS/Rashr#10]
Any chance to get support for Sony Xperia S (nozomi)?

**Comment by bastibearheyn** [comment]
Yes

**Comment by bastibearheyn** [comment]
Add me

**Comment by MichaelSp** [comment]
That reads like a "No" and I understand that. But can you give reasons? 
Not possible (technically)? 
No time? 
No test device?
Other reason?

**Comment by DsLNeXuS** [comment]
The first 3 options.. 
Sony Xperia S dosn't have a recovery partition. Sony uses a file? (Correct me if I'm wrong) placed in /system/recovery.tar  last year I've implemented the option to flash TWRP on Xperia S and T. The problem is not only the tar file needs to be changed, also the ric charger and chargermon needs to be edited. So because of reason 3 (no test device) I don't want to be responsible of bricked devices. Its easy to unbrick using Emma (Sony Flashtool) but not everybody know how to use it. So I decided to only implement the XZDualRecovery in Rashr (unfortunately only Xperia Z devices.)

I hope this reasons are enough and you are in the same opinion. 

Better safe and no Sony flashing then Bricks :)

PS: I know my answer is a little bit late :smile:
